docs/source/models/cause.rst

```diff
@@ -558,6 +558,95 @@ below.
 Remission Rates
 ^^^^^^^^^^^^^^^
 
+Definition
+""""""""""
+Remission is a measure of cases that recover from a with-condition state, given 
+a specified population and time period. Just as with incidence, the numerator is 
+given by the count of recovered (or *remitted*) cases, and the denominator is 
+the cumulative person-time [link to person-time definition in incidence 
+section] during which cases are able to go into remission:
+
+  .. math::
+    \frac{\text{number of remitted cases}}{\text{person-time in the with-condition 
+    population}}
+
+  For example, consider diarrhea cases in the Philippines in 2017. Say that that 
+  year, every such case remitted after an average of 5 days:
+
+  .. math::
+
+    \frac{\text{1 case}}{\text{5 person-days}}=\frac{365/5\text{ cases}}{\text{1 person-year}}=73\text{ cases/person-year}
+
+  This calculation is straightfoward, as diarrheal diseases have a highly
+  consistent disease duration.
+
+  In contrast, consider diabetes. Say that there were 142,794 prevalent cases of
+  diabetes (both type I and type II) in Moldova amongst males in 2017, and of 
+  those 142,794 cases, 509 remitted in 2017. This gives us the following rate:
+
+    .. math::
+      \frac{\text{509 cases}}{\text{142,794 person-years}} = \frac{\text{0.0036 cases}}{\text{1 person-year}}
+
+  It is important to note two things here: first, that this is a remission rate 
+  for diabetes at all ages, which obscures the generally increasing age-pattern 
+  that this rate follows. Second: there is no set duration for which one generally experiences diabetes. In fact, remission does not occur for type I, and is not guaranteed for type II. In the context of the diarrheal diseases example, this makes it seem as if diabetes cases remit, on average, after :math:`\frac{1}{0.0036}\simeq 279` years--which clearly does a poor job of capturing the behavior of diabetes. This sort of description was only 
+  appropriate for diarrhea, as there is a uniform remission rate across all 
+  cases. With diabetes, however, the remission rate does *not* tell us the 
+  average duration that an individual will experience diabetes.
+
+  How do we apply this to our simulants? Say we randomly selected 10 people with 
+  diarrhea in the Philippines on a random day in 2017. In the next day, they 
+  would accumulate 10 person days. Our rate tells us that in the next day, the 
+  expected value for cases remitted is given by:
+
+  .. math::
+
+    \frac{\text{1 case of diarrhea}}{\text{5 person-days}}\times\text{10 person-days} = \text{2 cases of diarrhea}.
```

Comment:
I would clarify in the equation portion that we are talking about 2 _remitted_ cases of diarrhea